lgc/patch/PatchInOutImportExport.cpp

```diff
@@ -758,33 +758,27 @@ void PatchInOutImportExport::visitCallInst(CallInst &callInst) {
       }
       case ShaderStageFragment: {
         unsigned interpMode = InOutInfo::InterpModeSmooth;
-        unsigned interpLoc = InOutInfo::InterpLocCenter;
         bool isPerPrimitive = false;
 
         if (!elemIdx)
           elemIdx = callInst.getOperand(isInterpolantInputImport ? 2 : 1);
         assert(isDontCareValue(elemIdx) == false);
 
-        Value *auxInterpValue = nullptr;
+        Value *interpValue = nullptr;
 
         if (isGenericInputImport) {
-          assert(callInst.arg_size() == 5);
-
-          isPerPrimitive = cast<ConstantInt>(callInst.getOperand(2))->getZExtValue() != 0;
-          interpMode = cast<ConstantInt>(callInst.getOperand(3))->getZExtValue();
-          interpLoc = cast<ConstantInt>(callInst.getOperand(4))->getZExtValue();
+          assert(callInst.arg_size() == 2);
+          isPerPrimitive = true;
```

Comment:
I think this needs a assignment to overwrite interpMode for per-primitive inputs.
interpMode = InOutInfo::InterpModeFlat

Reply:
Makes sense. It seems mesh shader tests are not yet enabled in our testing even partially?

Reply:
>  It seems mesh shader tests are not yet enabled in our testing even partially?

Yes, I request Vulkan driver team to enable part of the tests in our internal branch. Currently, we can only fetch the CTS containing mesh shader testing (VK.mesh_shader.ext.*) and do verification by ourselves.